### PR TITLE
New all needs nav

### PR DESCRIPTION
**Pull request by go8soft**
Task: https://trello.com/c/z0gNMAnI/144-new-all-triages-and-check-ins-navigation

### CODE REVIEW

File: app/views/shared/_side-nav.html.erb [hunk 1]
```diff
@@ -12,45 +12,38 @@
...
-              <%= link_to_unless_current "Team to do", root_path(:status => "to_do", :team => current_role_id), class: "main-menu__submenu-link" do %>
-                <span class="main-menu__submenu-current">Team to do</span>
-              <% end %>
-              <%= link_to_unless_current "Team in progress", root_path(:status => "in_progress", :team => current_role_id), class: "main-menu__submenu-link" do %>
-                <span class="main-menu__submenu-current">Team in progress</span>
-              <% end %>
-              <%= link_to_unless_current "Team completed", root_path(:status => "complete", :team => current_role_id), class: "main-menu__submenu-link" do %>
-                <span class="main-menu__submenu-current">Team Completed</span>
+          <% else %>
+              <%= link_to_unless_current "Support Actions", root_path(:status => "to_do"), class: "main-menu__link" do %>
```
Comment (rik86future) [inline app/views/shared/_side-nav.html.erb:23]: Needs

File: app/views/shared/_side-nav.html.erb [hunk 2]
```diff
@@ -12,45 +12,38 @@
...
+                <span class="main-menu__current">Support Actions</span>
```
Comment (rik86future) [inline app/views/shared/_side-nav.html.erb:24]: Needs

File: app/views/shared/_side-nav.html.erb (hunk 1 above)
Comment (go8soft) [reply app/views/shared/_side-nav.html.erb]: Thanks

File: app/views/shared/_side-nav.html.erb (hunk 2 above)
Comment (go8soft) [reply app/views/shared/_side-nav.html.erb]: Thanks

**Pull request merged**